README.md

```diff
@@ -44,3 +44,6 @@ See our [basic spring boot](./basic-spring-boot) example for a very simple refer
 ## Automated Deployments
 
 These pipeline quickstarts include an Ansible inventory through which they can be automatically deployed and managed using the [OpenShift Applier](https://github.com/redhat-cop/openshift-applier) role.
+
+## Optional: Use a container that contains ansible 
+So you don't have to install ansible on your machine.   Just type `oc run ansible-cli --image=openshift/origin-ansible  -- tail -f /dev/null`
```

Comment:
This container doesn't contain `git` -command so you couldn't clone this repository into the container.
So maybe also describe how to copy files into the `origin-ansible` container and how to access it for execution?

Reply:
I'm asking support and internal mailing lists if there is a better solution.

Reply:
@alberttwong @syvanen i dont feel that we need to explicitly call out this type of method in this repository. 

There is also a [toolbox](https://github.com/redhat-cop/containers-quickstarts/tree/master/tool-box) that performs similar functionality that you are looking for

Reply:
Can we update the readme that toolbox exists?  I didn't know that was the best way to deploy these scripts.